### ISSUE TITLE
JDBC gives inconsistent date values for their years 1600 or below 

### ISSUE THREAD

**Issue by njmarshall** [mapd/mapd-core#32]
### To reproduce the issue
1. Create a table and populate two dates including years of 1500 and 1600 on MAPDQL
CREATE TABLE t_date(id SMALLINT, val DATE);
INSERT INTO t_date VALUES(1, '1500-01-01');
INSERT INTO t_date VALUES(2, '1600-01-01');

2. Run the SQL stmt against MAPD
SELECT * FROM t_date ORDER BY id;

3.  Run the SQL stmt against MAPD thru JDBC
SELECT * FROM t_date ORDER BY id;

### Data Results from No. 2
 id|val
 1|1500-01-01
 2|1600-01-01

### Data Results from No. 3
 id|val
 1|1499-12-23
 2|1600-01-01

### Explanation
Noticeably, JDBC displaying the date for the row 1 is not the same as MAPD.  JDBC displays "1499-12-23" as supposedly matched with "1500-01-01" from MAPD DB.  Does this difference tell JDBC doing date validation than MAPD?

According to NOSQL's document, the range for the *Date* datatype is from 1000-01-01 to 9999-12-31.  Is that correct?

**Comment by njmarshall** [comment]
@dwayneberry Do you think the JDBC functionality for datatype displays should be identical to MAPD?

**Comment by dwayneberry** [comment]
@njmarshall we are in the weeds here.  Are you familiar with the 'Council of Trent'.  The calendar under went modifications in this time period to readjust.

**Comment by dwayneberry** [comment]
Sorry didn't mean to close it.

**Comment by dwayneberry** [comment]
I'm not sure where we would stand regarding UTC to pre "Council of Trent" definitely a conversation to be had over a beer :-)

**Comment by njmarshall** [comment]
Yes, my purpose is to find anything necessary to be fixed, improved or upgraded for rising a level of the software quality.
